raster/r.terraflow/types.h

```diff
@@ -72,7 +72,7 @@ typedef int cclabel_type;
 #define CCLABEL_FMT "%3d"
 /* the following are not arbitrary. LABEL_UNDEF should be the
  * most-negative value */
-static const cclabel_type LABEL_UNDEF=-1;
+static const cclabel_type LABEL_UNDEF = -1;
 static const cclabel_type LABEL_BOUNDARY=0;
 static const cclabel_type LABEL_NODATA=1;
 static const cclabel_type LABEL_START=1; /* the next label will be used */
```

Comment:
BTW, imho, these lines should also be formatted.

```suggestion
static const cclabel_type LABEL_BOUNDARY = 0;
static const cclabel_type LABEL_NODATA = 1;
static const cclabel_type LABEL_START = 1; /* the next label will be used */
```

Reply:
You're absolutely right, but it is out of scope of the aim for this PR.